homeassistant/components/dsmr_reader/definitions.py

```diff
@@ -91,34 +97,38 @@ def tariff_transform(value):
     },
     "dsmr/reading/phase_voltage_l1": {
         "name": "Current voltage L1",
-        "icon": "mdi:flash",
+        "device_class": DEVICE_CLASS_VOLTAGE,
         "unit": VOLT,
     },
     "dsmr/reading/phase_voltage_l2": {
         "name": "Current voltage L2",
-        "icon": "mdi:flash",
+        "device_class": DEVICE_CLASS_VOLTAGE,
         "unit": VOLT,
     },
     "dsmr/reading/phase_voltage_l3": {
         "name": "Current voltage L3",
-        "icon": "mdi:flash",
+        "device_class": DEVICE_CLASS_VOLTAGE,
         "unit": VOLT,
     },
     "dsmr/reading/phase_power_current_l1": {
         "name": "Phase power current L1",
-        "icon": "mdi:flash",
+        "device_class": DEVICE_CLASS_CURRENT,
         "unit": ELECTRICAL_CURRENT_AMPERE,
     },
     "dsmr/reading/phase_power_current_l2": {
         "name": "Phase power current L2",
-        "icon": "mdi:flash",
+        "device_class": DEVICE_CLASS_CURRENT,
         "unit": ELECTRICAL_CURRENT_AMPERE,
     },
     "dsmr/reading/phase_power_current_l3": {
         "name": "Phase power current L3",
-        "icon": "mdi:flash",
+        "device_class": DEVICE_CLASS_CURRENT,
         "unit": ELECTRICAL_CURRENT_AMPERE,
     },
+    "dsmr/reading/timestamp": {
```

Comment:
It's ok to add this value, but let's make sure that by default they are disabled in the entity registry by returning `False` from `entity_registry_enabled_default` property.

Reply:
I've added the `entity_registry_enabled_default` property together with the `enable_default` in the definitions.py.